packages/documentation/src/stories/templates/error-page/error-page.docs.mdx

```diff
@@ -0,0 +1,14 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import * as ErrorPageStories from './error-page.stories';
+
+<Meta of={ErrorPageStories} />
+
+<div class="container">
+  # Error page template
+
+  A generic error page pattern with an image. The image can be exchanged based on the type of error.
+
+  - 404 not found: <a download href="/images/content/404.png">Download "404 not found" image</a>
```

Comment:
The preceding text "404 not found:" is kind of irritating me. Wouldn't it be enough to just leave the text inside of the link?

Reply:
The example is very large and therefore (even if I now storybook probably pretty well) I was thinking about how to get the necessary code from the example.
It could be a good idea to add a note, and to tell visitors how to do this here.